safe/definitions/extra_keywords.py

```diff
@@ -82,6 +82,44 @@
     'description': None
 }
 
+extra_keyword_volcano_name = {
+    'key': 'volcano_name',
+    'name': tr('Volcano name'),
+    'description': tr('The name of the volcano.')
+}
+
+extra_keyword_eruption_height = {
+    'key': 'volcano_eruption_height',
+    'name': tr('Eruption height'),
+    'description': tr(
+        'The ash column height. It is calculated from the vent of the volcano '
+        'in metres.')
+}
+
+extra_keyword_volcano_eruption_event_time = {
+    'key': 'volcano_eruption_event_time',
+    'name': tr('Eruption event time'),
+    'description': tr(
+        'The time of the eruption. The data uploaded to the ash realtime is a '
```

Comment:
I would personally avoid using the word `realtime` because a lot of users are using InaSAFE all over the world without using/knowing the realtime platform specific to Indonesia. Is-it needed?

Reply:
Ah yes, sorry, copied from realtime. Fixed.